task/oracle-cloud-storage-download/0.2/samples/pipeline-download-directory.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: workspace-pvc
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: download-and-process
+spec:
+  params:
+  - name: bucket
+    description: OCI bucket name
+    type: string
+  - name: release-tag
+    description: Release tag to download
+    type: string
+  workspaces:
+  - name: shared
+    description: Workspace shared between tasks
+  - name: credentials
+    description: OCI credentials
+  tasks:
+  - name: download-from-oci
+    taskRef:
+      name: oracle-cloud-storage-download
```

Comment:
If we use the bundle resolver then applying the task step can be avoided and here you can use the bundle resolver 
WDYT ?